src/main/resources/application.properties

```diff
@@ -36,9 +36,10 @@ spring.data.elasticsearch.port=${ELASTIC_PORT:9300}
 spring.data.elasticsearch.searchport=${ELASTIC_SEARCHPORT:9200}
 spring.data.elasticsearch.drop=false
 spring.data.elasticsearch.executors=500
-spring.data.elasticsearch.executors.timeout=5
+spring.data.elasticsearch.executorsTimeout=5
```

Comment:
If we are changing existing property names then I suggest following changes for better readability and consistency:
```
spring.data.elasticsearch.executors.size
spring.data.elasticsearch.executors.timeout
spring.data.elasticsearch.reindex.size
spring.data.elasticsearch.reindex.timeout
```

Reply:
Done